src/CollectorGuess.cpp

```diff
@@ -69,6 +69,12 @@ bool isDouble(const std::string& x, LocaleInfo* pLocale) {
   return parseDouble(pLocale->decimalMark_, begin, end, res) && begin == end;
 }
 
+bool isTime(const std::string& x, LocaleInfo* pLocale) {
+  DateTimeParser parser(pLocale);
+  parser.setDate(x.c_str());
+  return parser.parse(pLocale->timeFormat_);
```

Comment:
Hmmmm, maybe this should default to `parseTime()`, since that's a more flexible default?